src/reducers/mcode.js

```diff
@@ -17,7 +17,7 @@ export default function mcode(state = defaultState, action) {
     if (action.type === types.INITIALIZE_SIMILAR_PATIENT_PROPS) {
         const { patient, condition } = action;
         const similarPatients = transformedTreatmentData.filter(treatmentDataPatient =>
-            isSimilarPatient(treatmentDataPatient, state.similarPatientProps));
+            isSimilarPatient(treatmentDataPatient, new FilterOptions(state.similarPatientProps)));
```

Comment:
This needs to be removed and call the actual OutcomesService, looks like it's left in as an oversight.